### PR TITLE
bootstrap-studio: init at 6.0.1

### PR DESCRIPTION
**Pull request by khrj**
###### Description of changes

https://bootstrapstudio.io/

"A powerful desktop app for creating responsive websites using the Bootstrap framework."

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/web/bootstrap-studio/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  name = "bootstrap-studio-${version}";
+  nameExecutable = pname;
+  src = fetchurl {
+    url = "https://bootstrapstudio.io/releases/desktop/${version}/Bootstrap%20Studio.AppImage";
+    sha256 = "sha256-piRqIB/bCF0IBZfXdlXrc7gmPTIUDbk8xhP7X5ozyWg=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit name src; };
+in
+appimageTools.wrapType2 {
+  inherit name src;
```
Comment (gytis-ivaskevicius) [inline pkgs/development/web/bootstrap-studio/default.nix:15]:
```suggestion
  inherit pname version src;
```

Comment (khrj) [reply pkgs/development/web/bootstrap-studio/default.nix:15]: I'm not how that would work, wrapType2 only takes a name, src, and extraPkgs?
image
[Source](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/appimage/default.nix)

**Comment by khrj** [comment]
Hey, it's been a couple of weeks - it'd be great if someone could take a look at this :) - @SuperSandro2000 could you please take a look (I'm not really sure whom to ping, but since you've reviewed a PR by me in the past)? Thanks!

File: pkgs/development/web/bootstrap-studio/default.nix (hunk 1 above)
Comment (khrj) [reply pkgs/development/web/bootstrap-studio/default.nix]: Ah I see, I missed the default value for name -- thank you, I've committed this

File: pkgs/development/web/bootstrap-studio/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{ lib, fetchurl, appimageTools }:
+
+let
+  pname = "bootstrap-studio";
+  version = "6.0.1";
+  name = "bootstrap-studio-${version}";
+  nameExecutable = pname;
```
Comment (SuperSandro2000) [inline pkgs/development/web/bootstrap-studio/default.nix:7]:
```suggestion
```

File: pkgs/development/web/bootstrap-studio/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  src = fetchurl {
+    url = "https://bootstrapstudio.io/releases/desktop/${version}/Bootstrap%20Studio.AppImage";
+    sha256 = "sha256-piRqIB/bCF0IBZfXdlXrc7gmPTIUDbk8xhP7X5ozyWg=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit name src; };
```
Comment (SuperSandro2000) [inline pkgs/development/web/bootstrap-studio/default.nix:12]:
```suggestion
  appimageContents = appimageTools.extractType2 { inherit pname version src; };
```

File: pkgs/development/web/bootstrap-studio/default.nix [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+in
+appimageTools.wrapType2 {
+  inherit pname version src;
+
+  extraInstallCommands = ''
+    mv $out/bin/${name} $out/bin/${pname}
```
Comment (SuperSandro2000) [inline pkgs/development/web/bootstrap-studio/default.nix:18]:
```suggestion
    mv $out/bin/${pname}-${version} $out/bin/${pname}
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**